.vsts.pipelines/builds/matrix.yml

```diff
@@ -23,16 +23,16 @@ jobs:
 
 - template: ../jobs/ci-linux.yml
   parameters:
-    job: debian82
-    imageName: microsoft/dotnet-buildtools-prereqs:debian-8.2-4e6e894-20181023212416
+    job: debian9
+    imageName: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-stretch-d61254f-20190807161114
```

Comment:
If new the Debian and Ubuntu source-build images I added work with 2.1 I think we should upgrade to those so we're consistent.